lib/routes/ustc/scms.ts

```diff
@@ -0,0 +1,94 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+import InvalidParameterError from '@/errors/types/invalid-parameter';
+
+const map = new Map([
+    ['kydt', { title: '中国科学技术大学化学与材料科学学院 - 科研动态', id: '2404' }],
+    ['tzgg', { title: '中国科学技术大学化学与材料科学学院 - 通知公告', id: '2402' }],
+    ['xshd', { title: '中国科学技术大学化学与材料科学学院 - 学术活动', id: 'xshd' }],
+    ['ynxw', { title: '中国科学技术大学化学与材料科学学院 - 院内新闻', id: '2405' }],
+]);
+
+const host = 'https://scms.ustc.edu.cn';
+
+export const route: Route = {
+    path: '/scms/:type?',
+    categories: ['university'],
+    example: '/ustc/scms/tzgg',
+    parameters: { type: '分类，见下表，默认为通知公告' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['scms.ustc.edu.cn/'],
+            target: '/scms',
+        },
+    ],
+    name: '化学与材料科学学院',
+    maintainers: ['boxie123'],
+    handler,
+    url: 'scms.ustc.edu.cn/',
+    description: `| 院内新闻 | 通知公告 | 科研动态 | 学术活动 |
+  | -------- | -------- | -------- | -------- |
+  | ynxw     | tzgg     | kydt     | xshd     |`,
+};
+
+async function handler(ctx) {
+    const type = ctx.req.param('type') ?? 'tzgg';
+    const info = map.get(type);
+    if (!info) {
+        throw new InvalidParameterError('invalid type');
+    }
+    const id = info.id;
+
+    const response = await got(`${host}/${id}/list.htm`);
```

Comment:
Consider allowing user to enter the id directly instead of requiring maintainers to update the mapping.